apps/src/util/whyDidYouRender.js

```diff
@@ -0,0 +1,22 @@
+import experiments from '@cdo/apps/util/experiments';
+import React from 'react';
+
+/**
+ * This experiment loads the why-did-you-render library to help track rendering
+ * performance issues: https://github.com/welldone-software/why-did-you-render
+ *
+ * To enable it for a specific component, add `static whyDidYouRender = true`
+ * at the top of the component class.
+ *
+ * Note: once we upgrade to React 16, we should upgrade this library.
+ */
+if (
+  process.env.NODE_ENV === 'development' &&
+  experiments.isEnabled(experiments.WHY_DID_YOU_RENDER)
+) {
+  const whyDidYouRender = require('@welldone-software/why-did-you-render');
+  whyDidYouRender(React, {
+    trackAllPureComponents: true,
+    trackHooks: false
+  });
+}
```

Comment:
This looks like it avoids running the new library in production mode, which is good. it does not exclude the library from the production webpack bundle, which is kind of bad in theory, but I was able to determine by using `yarn build:dist:analyze` that this only adds 100B to the production webpack bundle size. so I think we are ok here.